tool-plugins/intellij/build.gradle

```diff
@@ -40,11 +40,24 @@ repositories {
     maven { url 'http://dl.bintray.com/jetbrains/intellij-plugin-service' }
 }
 
-apply plugin: 'java'
+// LSP Dependencies
+dependencies {
+    compile group: 'org.eclipse.lsp4j', name: 'org.eclipse.lsp4j', version: '0.4.1'
+    compile group: 'io.get-coursier', name: 'coursier_2.12', version: '1.0.3'
+    compile group: 'io.get-coursier', name: 'coursier-cache_2.12', version: '1.0.3'
+    compile group: 'com.vladsch.flexmark', name: 'flexmark', version: '0.32.24'
+    compile 'org.scala-lang:scala-compiler:2.12.4'
```

Comment:
Is it possible to uniform these dependency declarations with the above ones?

Reply:
Yes it is, done